@navikt/core/react/src/form/combobox/combobox.stories.tsx

```diff
@@ -75,6 +76,52 @@ MultiSelect.args = {
   size: "medium",
 };
 
+export const MultiSelectWithComplexOptions: StoryFunction = (props) => {
+  const id = useId();
+  const [selectedOptions, setSelectedOptions] = useState<ComboboxOption[]>([]);
+  return (
+    <>
+      <UNSAFE_Combobox
+        id={id}
+        label="Komboboks - velg flere"
+        allowNewValues
+        options={props.options}
+        isMultiSelect={props.isMultiSelect}
+        onToggleSelected={(value, isSelected) =>
+          isSelected
+            ? setSelectedOptions([...selectedOptions, { label: value, value }])
+            : setSelectedOptions(
+                selectedOptions.filter((o) => o.value !== value),
+              )
+        }
+        size={props.size}
+      />
+      {selectedOptions.length > 0 && (
+        <dl>
+          {selectedOptions.map((option) => (
+            <React.Fragment key={option.label}>
+              <dt>{option.label}</dt>
+              <dd>{option.value}</dd>
+            </React.Fragment>
+          ))}
+        </dl>
+      )}
+    </>
+  );
+};
+
+MultiSelectWithComplexOptions.args = {
+  options: [
+    { label: "Apple", value: "apple" },
+    { label: "Banana", value: "banana" },
+    { label: "Orange", value: "orange" },
+    { label: "Pear", value: "pear" },
+    { label: "Grape", value: "grape" },
```

Comment:
For mer realistisk case burde kanskje value være noe helt annet enn label, feks. tall.

Reply:
Bra innspill! Har endret til temakoder (brukes for søknader for å rute til riktig team++).